activesupport/test/ordered_options_test.rb

```diff
@@ -36,14 +36,24 @@ def test_looping
     end
   end
 
-  def test_string_dig
+  def test_dig
     a = ActiveSupport::OrderedOptions.new
 
     a[:test_key] = 56
     assert_equal 56, a.test_key
     assert_equal 56, a["test_key"]
     assert_equal 56, a.dig(:test_key)
     assert_equal 56, a.dig("test_key")
+    assert_equal nil, a.dig(:non_existing_key)
```

Comment:
```suggestion
    assert_nil a.dig(:non_existing_key)
```

Reply:
Thanks for a review. I changed to `assert_nil` .